pkg/env/env.go

```diff
@@ -208,7 +208,8 @@ func (e *testEnv) processTestActions(ctx context.Context, t *testing.T, actions
 func (e *testEnv) processTestFeature(ctx context.Context, t *testing.T, featureName string, feature types.Feature) context.Context {
 	skipped, message := e.requireFeatureProcessing(feature)
 	if skipped {
-		t.Skipf(message)
+		t.Log(message)
+		return ctx
```

Comment:
This will not mark the test as `SKIP`'ed correct ? The intent of `skip` here was to make it clear that the test was skipped. Returning ctx will not mark the test as skipped. Can be misleading.